lib/plugins/console/index.ts

```diff
@@ -1,4 +1,6 @@
-export = function(ctx) {
+import Hexo from '../../hexo';
```

Comment:
```suggestion
import type Hexo from '../../hexo';
```

Use `import type` for type only imports.

Reply:
Fixed in 3ad806d.

Maybe we should add a rule in `.eslintrc.json` to enforce this style?

```json
{
  "rules": {
    "@typescript-eslint/consistent-type-imports": "warn",
  }
}
```